src/NluEngine/Component/NluComponentInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace OpenDialogAi\NluEngine\Component;
+
+use OpenDialogAi\AttributeEngine\CoreAttributes\UtteranceAttribute;
+use OpenDialogAi\InterpreterEngine\Interpreters\AbstractNLUInterpreter\AbstractNLURequestFailedException;
+use OpenDialogAi\NluEngine\Response\ResponseInterface;
+
+interface NluComponentInterface
```

Comment:
Let's rename these classes after "LanguageService" rather than "Nlu".

```suggestion
interface LanguageServiceComponentInterface
```